peter_rockwood/memory-cards.js

```diff
@@ -0,0 +1,50 @@
+var MemoryCards = (function() {
```

Comment:
Looks good! The only code issue I've got for you comes when you complete a match&mdash;I've noted it below.